commonmark/en/content/metadata/Cause_of_death.md

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
```

Comment:
This is not a markdown file. It is HTML. 
It also seems to be in the wrong place. I presume it should be removed?